src/main/resources/add-ons-to-index.json

```diff
@@ -614,7 +635,7 @@
       "type": "OMOD",
       "name": "Database Backup",
       "tags": [
-        "administration"
+        "administration", "Database"
```

Comment:
"database" is not a helpful tag.

Reply:
@f4ww4z are you sure it is not helpful for a database backup module?

Reply:
@dkayiwa Yes, I am very sure as @djazayeri mentions this in another PR. See https://github.com/openmrs/openmrs-contrib-addonindex/pull/27#discussion_r154514608

Reply:
That means you need to explain why. Just like @djazayeri did.

Reply:
Note taken, thanks @dkayiwa . @Yashu-Jindal Remove "database" since grouping modules with the tag doesn't suggest a useful purpose.

Reply:
@f4ww4z you comment is still slightly different from that of Darius 😊

Reply:
@dkayiwa I edited it